<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>db_helper</class>
 <widget class="QMainWindow" name="db_helper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>410</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>DB_Helper</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>501</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_1">
     <widget class="QLineEdit" name="dbName_lineEdit">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>150</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>130</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Database name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="link_lineEdit">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>150</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="password_lineEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>150</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>130</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Link</string>
      </property>
     </widget>
     <widget class="QPushButton" name="setConnection_pushButton">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>180</y>
        <width>481</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Set connection</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="login_lineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLineEdit" name="users_upperBalanceLimit_lineEdit">
      <property name="geometry">
       <rect>
        <x>152</x>
        <y>160</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>110</y>
        <width>471</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>USERS</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="transactions_upperTransactedLimit_lineEdit">
      <property name="geometry">
       <rect>
        <x>152</x>
        <y>300</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Entry amount</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>338</x>
        <y>280</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Lower transacted limit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>140</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Upper balance limit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>338</x>
        <y>140</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Lower balance limit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>280</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Upper transacted limit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="transactions_createEntries_pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>481</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Create entries</string>
      </property>
     </widget>
     <widget class="QPushButton" name="dropDatabase_pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>481</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Drop database</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Entry amount</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="transactions_lowerTransactedLimit_lineEdit">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>300</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="transactions_entryAmount_lineEdit">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>300</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>250</y>
        <width>471</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TRANSACTIONS</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="users_createEntries_pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>481</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Create entries</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="users_entryAmount_lineEdit">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>160</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="createTables_pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>481</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Create tables</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="users_lowerBalanceLimit_lineEdit">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>160</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
